opossum-tool/OpossumUI
# Add menu item for QA mode

### Summary of changes

This commit adds a new entry to the menu to select the new QA mode. If the mode is selected the icon will be a checked checkbox if not an unchecked one.
I decided to put the new QA item in the submenu of View as this different mode feels like a View property. I think, one could also put it in the Edit submenu, I don't have a strong opinion here.
### Context and reason for change

Selecting preferred attributions should only be possible in the new QA mode. As a first step this commit adds the respective menu item.

I downloaded the icons from [here](https://fonts.google.com/icons?icon.set=Material+Icons&selected=Material+Icons+Outlined:search:). 
### How can the changes be tested

Check out the changes of the PR, open OpossumUI observe the unchecked checkbox next to the QA menu item as below
![Screenshot from 2023-10-04 14-40-50](https://github.com/opossum-tool/OpossumUI/assets/50019307/43df8e8e-3160-423d-aaa5-a9efa90d7bb0)
Click the menu item, reopen the submenu and observe the different icon as below
![Screenshot from 2023-10-04 14-41-40](https://github.com/opossum-tool/OpossumUI/assets/50019307/b1f54880-890e-4b13-a279-3e0f733775e1)
